plugin/db/sqlite/sqlite.go

```diff
@@ -599,6 +596,12 @@ func (driver *Driver) Dump(ctx context.Context, database string, out io.Writer,
 	return nil
 }
 
+type sqliteSchema struct {
+	stype     string
```

Comment:
stype is ambigous sqlType? statementType? is it possible to define as ENUM?

Reply:
Since it's queried from sqlite, we may have to use string. Changed to schemaType instead.